src/overlays/actors/ovl_En_S_Goro/z_en_s_goro.h

```diff
@@ -5,17 +5,157 @@
+    EnSGoro_Msg_WShrineA_Goron_GKLoud_02,
+    EnSGoro_Msg_WShrineA_Goron_GKLoud_03, //Also TalkAgain
+    EnSGoro_Msg_WShrineA_NonGoron_GKQuiet_00,
+    EnSGoro_Msg_WShrineA_NonGoron_GKQuiet_01,
+    EnSGoro_Msg_WShrineA_TalkAgain_NonGoron_GKQuiet,
+    EnSGoro_Msg_WShrineA_Goron_GKQuiet_00,
+    EnSGoro_Msg_WShrineA_Goron_GKQuiet_01,
+    EnSGoro_Msg_WShrineA_TalkAgain_Goron_GKQuiet,
+    EnSGoro_Msg_WShrineA_TorchOkay_00,
+    EnSGoro_Msg_WShrineA_TorchOkay_01,
+    EnSGoro_Msg_WShrineA_TalkAgain_TorchOkay,
+    EnSGoro_Msg_WShrineC_NonGoron_GKLoud,
+    EnSGoro_Msg_WShrineC_Goron_GKLoud_00,
+    EnSGoro_Msg_WShrineC_Goron_GKLoud_01,
+    EnSGoro_Msg_WShrineC_Goron_GKLoud_02,
+    EnSGoro_Msg_WShrineC_Goron_GKLoud_03,
+    EnSGoro_Msg_WShrineC_TalkAgain_Goron_GKLoud,
+    EnSGoro_Msg_WShrineC_Goron_GKQuiet_00,
+    EnSGoro_Msg_WShrineC_Goron_GKQuiet_01,
+    EnSGoro_Msg_WShrineC_Goron_GKQuiet_02,
+    EnSGoro_Msg_WShrineC_TalkAgain_Goron_GKQuiet,
+    EnSGoro_Msg_WShrineC_NonGoron_GKQuiet,
+    EnSGoro_Msg_WShrineB_NonGoron_GKLoud,
+    EnSGoro_Msg_WShrineB_TalkAgain_NonGoron_GKLoud,
+    EnSGoro_Msg_WShrineB_Goron_GKLoud_00,
+    EnSGoro_Msg_WShrineB_Goron_GKLoud_01,
+    EnSGoro_Msg_WShrineB_Goron_GKLoud_02, //Also talkagain
+    EnSGoro_Msg_WShrineB_NonGoron_GKQuiet_00,
+    EnSGoro_Msg_WShrineB_NonGoron_GKQuiet_01,
+    EnSGoro_Msg_WShrineB_NonGoron_GKQuiet_02,
+    EnSGoro_Msg_WShrineB_NonGoron_GKQuiet_03,
+    EnSGoro_Msg_WShrineB_NonGoron_GKQuiet_04,
+    EnSGoro_Msg_WShrineB_TalkAgain_NonGoron_GKQuiet,
+    EnSGoro_Msg_WShrineB_Goron_GKQuiet_00,
+    EnSGoro_Msg_WShrineB_Goron_GKQuiet_01,
+    EnSGoro_Msg_WShrineB_Goron_GKQuiet_02,
+	EnSGoro_Msg_WShrineB_TalkAgain_Goron_GKQuiet,
+    EnSGoro_Msg_SShrineA_NonGoron = 0xde1,
+    EnSGoro_Msg_SShrineA_Goron_JGWaiting,
+    EnSGoro_Msg_SShrineA_Goron_JGTalked_00,
+    EnSGoro_Msg_SShrineA_Goron_JGTalked_01,
+    EnSGoro_Msg_SShrineB_NonGoron,
+    EnSGoro_Msg_SShrineB_Goron_JGWaiting,
+    EnSGoro_Msg_SShrineB_Goron_JGTalked_00,
+    EnSGoro_Msg_SShrineB_Goron_JGTalked_01,
+    EnSGoro_Msg_SShrineC_NonGoron,
+    EnSGoro_Msg_SShrineC_Goron_JGWaiting,
+    EnSGoro_Msg_SShrineC_Goron_JGTalked_00,
+    EnSGoro_Msg_SShrineC_Goron_JGTalked_01,
+    EnSGoro_Msg_SShrineD_NonGoron_00,
+    EnSGoro_Msg_SShrineD_NonGoron_01,
+    EnSGoro_Msg_SShrineD_NonGoron_02,
+    EnSGoro_Msg_SShrineD_Goron_JGWaiting,
+    EnSGoro_Msg_SShrineD_Goron_JGTalked_00,
+    EnSGoro_Msg_SShrineD_Goron_JGTalked_01,
+    EnSGoro_Msg_SShrineE_NonGoron_00,
+    EnSGoro_Msg_SShrineE_NonGoron_01,
+    EnSGoro_Msg_SShrineE_Goron_JGWaiting,
+    EnSGoro_Msg_SShrineE_Goron_JGTalked_00,
+    EnSGoro_Msg_SShrineE_Goron_JGTalked_01,
+    EnSGoro_Msg_SShrineF_NonGoron_00,
+    EnSGoro_Msg_SShrineF_NonGoron_01,
+    EnSGoro_Msg_SShrineF_Goron_JGWaiting,
+    EnSGoro_Msg_SShrineF_Goron_JGTalked_00,
+    EnSGoro_Msg_SShrineF_Goron_JGTalked_01,
+    EnSGoro_Msg_SShrineF_Goron_JGTalked_02
+} EnSGoro_MessageId;
```

Comment:
Currently we just use raw HEX values for textIds. there is an idea to use defines (#586) , but we are not sure yet

Reply:
And we wouldn't do it like this even if we did use them now, they'd be kept centrally in case they're needed in several places.